packages/11ty/CHANGELOG.md

```diff
@@ -24,14 +24,18 @@ Changelog entries are classified using the following labels:
 
 - Validation method to check if an image can be tiled, and log error if invalid.
 
-### Removed
-
-- Remove `.jp2` from supported image extensions
-
 ### Changed
 
 - Increase `print-image` transformation width to `2025px`
 
+### Fixed
+
+- Sort epub pages for reading order by sort key rather than filename. Fixes epub validation error for filenames that begin with numbers.
```

Comment:
```suggestion
- Sort epub pages for reading order by sort key rather than filename. Fixes epub validation error for filenames that begin with numbers.
- Prefix epub filename with `page-` to ensure validity if filename begins with a number
```